scalableminds/webknossos
# Catch exceptions in FileSystemDataVault, return Empty or Failure

### Steps to test:
- (I tested this already locally)
- in local zarr dataset, delete a chunk
- see that no exceptions are logged, and the fill value is displayed in the frontend
- when read permissions are revoked, rate-limited logging should show, but no bare exceptions

### Issues:
- fixes #7210

------
